modules/gltf/gltf_document.cpp

```diff
@@ -5608,6 +5608,19 @@ void GLTFDocument::_generate_scene_node(Ref<GLTFState> p_state, Node *scene_pare
 		// We change the scene_parent to our bone attachment now. We do not set current_node because we want to make the node
 		// and attach it to the bone_attachment
 		scene_parent = bone_attachment;
+	} else if (gltf_node->skin >= 0 && gltf_node->mesh >= 0) {
+		Node3D *node_3d = _generate_spatial(p_state, node_index);
+
+		scene_parent->add_child(node_3d, true);
+		node_3d->set_owner(scene_root);
+
+		node_3d->set_transform(gltf_node->xform);
+		node_3d->set_name(_gen_unique_name(p_state, node_3d->get_name()));
```

Comment:
@lyuma do you prefer another naming convention?

Reply:
I think `gltf_node->get_name()` has already been uniquified, and it's okay for them to share the same name since it's guaranteed one will be a child of the other, not siblings. So I'll use `mesh_inst->set_name(gltf_node->get_name());`